dcos/config.py

```diff
@@ -323,9 +323,14 @@ def get_config_schema(command):
             pkg_resources.resource_string(
                 'dcos',
                 'data/config-schema/core.json').decode('utf-8'))
-
-    executable = subcommand.command_executables(command)
-    return subcommand.config_schema(executable, command)
+    elif command in subcommand.default_subcommands():
+        return json.loads(
+            pkg_resources.resource_string(
+                'dcoscli',
+                'data/config-schema/{}.json'.format(command)).decode('utf-8'))
```

Comment:
@tamarrow this code is in the `dcos` package and doesn't have access to the `dcoscli` package's data files. If we want to do this then we should move the data files to the `dcos` package.